test/active_record_test.rb

```diff
@@ -179,28 +177,18 @@ def test_should_set_attributes_regardless_of_arguments_order
     Account.new.attributes = { password: "password", key: SECRET_KEY }
   end
 
-  def test_should_create_changed_predicate
-    person = Person.create!(email: 'test@example.com')
-    refute person.email_changed?
-    person.email = 'test@example.com'
-    refute person.email_changed?
-    person.email = nil
-    assert person.email_changed?
-    person.email = 'test2@example.com'
-    assert person.email_changed?
-  end
-
   def test_should_create_was_predicate
     original_email = 'test@example.com'
     person = Person.create!(email: original_email)
-    assert_equal original_email, person.email_was
```

Comment:
I chose to remove these assertions as the behavior seems broken, or at least surprising, in AR < 5.2. If any code relies on `*_was` being set to the current value after a model has been created this update will break that behavior. It's understandable if this PR is blocked on carrying the old behavior forward but I would like others to weigh in.

Reply:
It looks like the previous `*_was` behavior is still the behavior in AR 5.2, so I'll look into carrying that behavior over.